metrics/firebase/functions/test/main_test.dart

```diff
@@ -16,65 +16,69 @@ import 'test_utils/test_data/build_test_data_generator.dart';
 // ignore_for_file: avoid_redundant_argument_values
 
 void main() {
-  group("onBuildAddedHandler", () {
-    const tasksCollectionName = 'tasks';
-    const projectId = 'projectId';
-    const durationInMilliseconds = 123;
-    const buildStatus = BuildStatus.successful;
-
-    final startedAt = DateTime.now();
-    final startedAtDayUtc = startedAt.toUtc().date;
-    final testDataGenerator = BuildTestDataGenerator(
-      projectId: projectId,
-    );
-
-    final firestoreMock = FirestoreMock();
-    final collectionReferenceMock = CollectionReferenceMock();
-    final taskCollectionReferenceMock = CollectionReferenceMock();
-    final documentReferenceMock = DocumentReferenceMock();
-    final documentSnapshotMock = DocumentSnapshotMock();
-
-    PostExpectation<Firestore> whenFirestore() {
-      return when(documentSnapshotMock.firestore);
-    }
-
-    PostExpectation<DocumentReference> whenDocument() {
-      whenFirestore().thenReturn(firestoreMock);
+  const tasksCollectionName = 'tasks';
+  const projectId = 'projectId';
+  const durationInMilliseconds = 123;
+  const buildStatus = BuildStatus.successful;
+
+  final startedAt = DateTime.now();
+  final startedAtDayUtc = startedAt.toUtc().date;
+  final testDataGenerator = BuildTestDataGenerator(
+    projectId: projectId,
+  );
+  final buildJson = testDataGenerator.generateBuildJson();
+  final buildJsonDocumentData = DocumentData.fromMap(buildJson);
+
+  final firestoreMock = FirestoreMock();
+  final collectionReferenceMock = CollectionReferenceMock();
```

Comment:
```suggestion
  final collectionReference = CollectionReferenceMock();
```
What do you think? Please, apply this suggestion to all similar variables.